packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -45,6 +46,27 @@ import {
 import { SegmentGroupCollection } from "./segmentGroupCollection";
 import { PropertiesManager, PropertiesRollback } from "./segmentPropertiesManager";
 
+// TODO: Once @fluid-internal/attributor is made public, this package should reference that type.
```

Comment:
Just a note. If this interface will be the one that crosses layers, it will probably go into runtime definitions, as it'll break layers to depend on the implementations necessary to implement attributor.